homeassistant/components/fastdotcom/sensor.py

```diff
@@ -64,13 +58,13 @@ async def async_added_to_hass(self):
             return
         self._state = state.state
 
-    def update(self):
+    def update(self) -> None:
         """Get the latest data and update the states."""
-        data = self.speedtest_client.data
+        data = self._speedtest_data.data  # type: ignore[attr-defined]
         if data is None:
             return
         self._state = data["download"]
```

Comment:
you can set the state here directly so you don't need the state property:
`self._attr_state = data["download"]`